### ISSUE TITLE
aggressive-indent in Clojure(Script) mode renders Emacs unresponsive

### ISSUE THREAD

**Issue by neshtea** [syl20bnr/spacemacs#9456]
#### Description :octocat:
Editing Clojure(Script) files and saving them, CPU load goes to 100% for a considerable amount of time (about 60s).

#### Reproduction guide :beetle:
- Start Emacs
- add the `clojure` layer to `.spacemacs`
- In `user-config`, add `aggressive-indent` to `clojure-mode-hook`
- Open a `.cljs` file, make some changes, save the file.

*Observed behaviour:* :eyes: :broken_heart:
CPU load goes to 100% for a considerable amount of time (about 60s), rendering Emacs unresponsive for this time.

*Expected behaviour:* :heart: :smile:
Basically the opposite of the observed behaviour.

#### System Info :computer:
- OS: darwin
- Emacs: 25.2.1
- Spacemacs: 0.200.9
- Spacemacs branch: master (rev. 8e1af145)
- Graphic display: t
- Distribution: spacemacs
- Editing style: vim
- Completion: helm
- Layers:
```elisp
(elixir clojure helm auto-completion emacs-lisp git markdown org spell-checking syntax-checking version-control)
```
- System configuration features: NOTIFY ACL GNUTLS LIBXML2 ZLIB TOOLKIT_SCROLL_BARS NS MODULES

#### Backtrace :paw_prints:
```
<<BACKTRACE IF RELEVANT>>
```

**Comment by kommen** [comment]
I'm using aggressive-indent mode a lot with Clojure and ClojureScript, and didn't have any problems with it recently. Is this still an issue for you?

**Comment by neshtea** [comment]
I did not notice this behaviour for a while, so no, these problems seem to have vanished for me.

**Comment by d12frosted** [comment]
Ok, thanks everyone for checking this.

I am closing this issue, but feel free to reopen if you encounter this problem again.
